common/src/main/java/com/wynntils/utils/mc/StyledTextUtils.java

```diff
@@ -42,4 +43,12 @@ public static Optional<Location> extractLocation(StyledText text) {
                 Integer.parseInt(matcher.group(2)),
                 Integer.parseInt(matcher.group(3))));
     }
+
+    public static StyledText joinLines(List<StyledText> lines) {
+        String description = String.join(
+                        " ", lines.stream().map(StyledText::getString).toList())
+                .replaceAll("\\s+", " ")
+                .trim();
+        return StyledText.fromString(description).getNormalized();
```

Comment:
Why normalize? It seems like a weird (and perhaps unintended) side effect of join.